src/main/java/io/xpring/xpring/XpringClient.java

```diff
@@ -0,0 +1,60 @@
+package io.xpring.xpring;
+
+import io.xpring.payid.PayIDClientInterface;
+import io.xpring.payid.PayIDException;
+import io.xpring.xrpl.Wallet;
+import io.xpring.xrpl.XRPClientInterface;
+import io.xpring.xrpl.XRPException;
+
+import java.math.BigInteger;
+
+/**
+ * Composes interactions of Xpring services.
+ */
+public class XpringClient {
+  /**
+   * A {@link PayIDClientInterface} used to interact with the Pay ID protocol.
+   */
+  @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+  private PayIDClientInterface payIDClient;
+
+  /**
+   * A {@link XRPClientInterface} used to interact with the XRP Ledger protocol.
+   */
+  private XRPClientInterface xrpClient;
```

Comment:
Same here - it's bad form in Java to have suffixes like "Interface" because they're redundant.
```suggestion
  private XRPClient xrpClient;
```